tests/lib/rules/newline-per-chained-call.js

```diff
@@ -105,6 +113,35 @@ ruleTester.run("newline-per-chained-call", rule, {
             "    // Do something with error.",
             "}).end();"
         ].join("\n"),
+        output: [
+            "http.request({",
+            "    // Param",
+            "    // Param",
+            "    // Param",
+            "}).on('response', function(response) {",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "    // Do something with response.",
+            "})\n.on('error', function(error) {",
```

Comment:
Nitpick: Since we're actually writing this out with whitespace I think it would be clearer to add a new element to the array instead of inlining a `\n` character.

For example:

```
[
  "})",
  ".on('error', function(error) {",
]
```